building-with-ocean/asset-hosting.md

```diff
@@ -17,134 +17,136 @@ On Ocean Marketplace, a publisher must provide the link to the asset during publ
 
 Publishers can choose any hosting service of their choice. The below section explains how to use commonly used hosting services with Ocean Marketplace.
```

Comment:
Please search for all mentions of GDrive and remove it. Example: "For example, a publisher can store the content on their Google Drive, AWS serve".